doc/lib_enum.md

```diff
@@ -1,32 +1,30 @@
-// This file is distributed under the terms of the MIT license, (c) the KSLib team
-
 ## lib_enum
 
 ``lib_enum.ks`` provides a set of functions for manipulating lists, queues, and stacks using the new delegates syntax introduced in kOS version 0.19.0. It allows you to transform and execute on lists, queues, and stacks by passing in a delegate function, and is designed to contain all the core components of an enumerable library.
 
 Each of the functions is made available through the `Enum` [lexicon](http://ksp-kos.github.io/KOS_DOC/structures/misc/lexicon.html), to minimize the number of variables used in the global namespace. They can be called using the following syntax: `run lib_enum. Enum["max"](list(1,2,3)). // 3` All functions work on [Lists](http://ksp-kos.github.io/KOS_DOC/structures/misc/list.html), [Stacks](http://ksp-kos.github.io/KOS_DOC/structures/misc/stack.html), and [Queues](http://ksp-kos.github.io/KOS_DOC/structures/misc/queue.html), and when collections are returned, they will be of the same type as the original input.
 
-function        | arguments                     | return type   | description
```

Comment:
I don't know why diff called these out.  I did no deliberate edits here and did no search & replace that could have gotten it either. When rendered, both on Github and my markdown previewer, it looks unchanged.  In other words, "I didn't change anything," and "it works on my machine."  How many times have I heard THOSE words...